src/styles/index.styl

```diff
@@ -1,4 +1,4 @@
-@require 'cozy-ui/build'
+@import '~cozy-ui/dist/cozy-ui.min.css'
```

Comment:
Taken from the updated template https://github.com/CPatchane/create-cozy-app/blob/master/packages/cozy-scripts/template/app/src/styles/index.css#L2

Not ideal though as I think we have duplicated styles.

Reply:
Why duplicated styles? Maybe we could compare the bundle sizes?

Reply:
I'm absolutely 100% positive we're having duplicated styles with this but, indeed, that's the only way to have erverything working so far.

Reply:
Yep the bundle size probably exploded, but the alternate solution is not clear to me — see https://github.com/cozy/cozy-ui/issues/866